src/test/java/core/basesyntax/MyTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class MyTest {
+    private User user;
+    private String login;
+    private int loginLength;
+    private String password;
+    private int age;
+    private int passwordLength;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        login = user.getLogin();
+        password = user.getPassword();
+        age = user.getAge();
+        loginLength = user.getLogin().length();
+        passwordLength = user.getPassword().length();
+    }
+
+    @Test
+    void user_notNull_check() {
+        assertNotNull(user, "User is invalid!" );
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        assertNotNull(login);
+    }
+
+    @Test
+    void login_length_notZero_check() {
+        assertThrows(MyUncheckedException.class , () -> {
+            if (loginLength == 0) {
+                throw new MyUncheckedException("Login should be bigger!");
+            }
+        });
```

Comment:
The use of `assertThrows` is correct, but ensure that the logic inside the lambda expression accurately reflects the application's requirements for login length validation.